kuma/wiki/models.py

```diff
@@ -1663,6 +1613,7 @@ class Revision(models.Model):
 
     summary = models.TextField()  # wiki markup
     content = models.TextField()  # wiki markup
+    tidied_content = models.TextField(blank=True)  # wiki markup tidied up
```

Comment:
Doh, sorry I over-looked that; I jumped straight to this `models.py` diff to see what you did here.